src/physics/em/detail/UrbanMscStepLimit.hh

```diff
@@ -137,14 +167,14 @@ CELER_FUNCTION auto UrbanMscStepLimit::operator()(Engine& rng) -> MscStep
 {
     MscStep result;
 
-    result.phys_step = helper_.step_length();
-    result.true_path = min<real_type>(result.phys_step, range_);
+    result.phys_step = phys_step_;
+    result.true_path = phys_step_;
```

Comment:
Same comment as above - the current code may be more general (?) to support any special cases.

Reply:
Let's focus on the specific for now and generalize if needed.